static/js/tools/choropleth/choropleth.tsx

```diff
@@ -371,7 +371,7 @@ function determineColorPalette(dict, pc: boolean, popMap: []): number[] {
   });
   const len = values.length;
   const lowerValue = values[0];
-  const medianValue = values[len / 2];
+  const medianValue = values[Math.floor(len / 2)];
```

Comment:
Median is defined as the middle value if there is an odd number of values.  It is the average of the two middle values if there are an even number of values.  It is not defined if there are no values.